src/main/java/br/unb/cic/analysis/AbstractMergeConflictDefinition.java

```diff
@@ -17,18 +19,49 @@
 public abstract class AbstractMergeConflictDefinition {
     protected List<Statement> sourceStatements;
     protected List<Statement> sinkStatements;
+    protected List<Statement> inBetweenStatements;
 
     public AbstractMergeConflictDefinition() {
         sourceStatements = new ArrayList<>();
         sinkStatements = new ArrayList<>();
+        inBetweenStatements = new ArrayList<>();
```

Comment:
@rafaelmotaalves what do you mean by inBetweenStatements? and why you need this? is this the same as baseStatements? do you consider that we can have left; base; right; base; left and so on?

Reply:
In between was those that were neither source nor sink, but this approach didn't work out because I couldn't mark statements outside from the source method. I forgot to remove this attribute, will remove it. @pauloborba